src/charts/BarChart.tsx

```diff
@@ -5,20 +5,22 @@ import { useNetworkGet } from '../network';
 import { useRefresh } from '../context/RefreshContext';
 
 export default function BarChart(props: BarChartProps) {
-  const [chartData, setChartData] = useState<StatsData>({ years: [], cites: [] });
+  const [chartData, setChartData] = useState<StatsData>({ years: [], counts: [] });
   const refresh = useRefresh();
+  const labelColors = chartData.counts.map((value) => (value > 0 ? 'rgb(55, 61, 63)' : '#b6b6b6'));
 
   const refetch = useNetworkGet(`fe/${props.route}/stats`, 'statsData', (data: StatsData) => {
     setChartData(data);
   });
+
   useEffect(() => {
     refresh.addRefetch(refetch);
   }, []);
 
   const series = [
     {
-      name: 'citations',
-      data: chartData.cites,
+      name: '#' + props.yDimension,
```

Comment:
What is the "#" for?

Reply:
So the label says "#citations". This means "Amount of citations". The long version is shown on the y axis, while this short version is shown in the box, while you hover over a bar.